src/vmm/src/guest_config/aarch64_templates/mod.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::BTreeMap;
+
+use arch::aarch64::regs::Error;
+use kvm_ioctls::VcpuFd;
+
+/// Module with static aarch64 template
+pub mod g3g2;
+
+/// Structure that represents change in the register
+/// We zero bits with zero mask and set with values mask
```

Comment:
This is a bit confusing. Could we rephrase?

Reply:
I can try to rephrase, but the problem is... we still can not agree on one way to use these `mask` and `value`. There are 3 or 4 proposals and I just use the one I thought of first. Also this `RegisterValue` struct technically should be in `common` types somewhere.